react-native.config.js

```diff
@@ -5,7 +5,7 @@ module.exports = {
 				project: './platforms/ios/SQLite.xcodeproj'
 			},
 			android: {
-				sourceDir: './platforms/android'
+				sourceDir: './platforms/android-native'
```

Comment:
so why are you changing this to android-native? ISn't this going to break for people wanting to use regular Android ?

Reply:
We were using android-native in our project. I'll change it back to android.